matrix-arithmetic.Rmd

```diff
@@ -92,20 +66,110 @@ $$
 C = -A
 $$
 
-### Derivatives {-}
+### Tangent {-}
 
 $$
-\frac{\partial}{\partial A} C = -1
+\dot{C} = -\dot{A}
+$$
+
+### Adjoint {-}
+
+$$
+\overline{A} \ \ {\small +}{=} \ \ -\overline{C}
+$$
+
+## Inverse
+
+$$
+C = A^{-1}
 $$
 
 ### Tangent {-}
 
 $$
-\dot{C} = -\dot{A}
+\dot{C} = -C \cdot \dot{A} \cdot C
 $$
 
 ### Adjoint {-}
 
 $$
-\overline{A} \ \ {\small +}{=} \ \ -\overline{C}
+\overline{A} \ \ {\small +}{=} \ \ -C^T \cdot \overline{C} \cdot C^T
+$$
+
+## Determinant
+
+$$
+C = \det{A}
+$$
+
+### Tangent {-}
+
+$$
+\dot{C} = C \cdot \textrm{Tr} \left( A^{-1} \dot{A}\right)
+$$
+
+### Adjoint {-}
+
+$$
+\overline{A} \ \ {\small +}{=} \ \ \overline{C} \cdot C \cdot \left( A^{-1} \right)^T
+$$
+
+## Matrix inverse product
+
+$$
+C = A^{-1} \cdot B
```

Comment:
I'd prefer these to be listed as left and right division.  I'd just change the titles to `Matrix left division` and `Matirx right division` and give the full definitions as

```
C = A \ B = A^{-1} \cdot B
```

and 

```
C = A / B = A \cdot B^{-1}
```

Reply:
No problem.

Reply:
Added right division. It's not on Giles so I did the math myself; I think it's alright but an extra pair of eyes is probably a good idea. :-)

Reply:
If we were real mathematicians, we'd just note 
```
A / B = (B' \ A')'
```
and be done with it.  I'd prefer to write out the answer.

Reply:
Thanks for adding!  I'll try to verify the above.